README.rst

```diff
@@ -14,6 +14,37 @@ contributions!
 See the `developer docs`_ and the `contribution docs`_ for more information.
 
 
+Tool Power User
+===============
+
+At Crate.io we are defining so called Power Users to ensure the integrations keep working.
+
+Responsiblities of Power Users:
+
+* Keep track of releases, updates and developments of the tools.
+* Test integration after minor and mayor CrateDB releases.
+* Test integration after tool releases.
+* Have a working setup available including a quick demo.
+* Being able to answer questions for the specific tool.
+* Ensuring guides are up to date.
+
+
+Power Users (*Tool, User, Tool Version // CrateDB Version, Issues*) :
+------------
```

Comment:
```suggestion
Power Users (*Tool, User, Tool Version // CrateDB Version, Issues*) :
---------------------------------------------------------------------
```

Reply:
however, I would drop this from the title and instead reformat the list as a table

Reply:
tried to avoid that. :)

will do.